docs/source/traitsui_user_manual/testing.rst

```diff
@@ -41,6 +41,7 @@ Discussions
    :maxdepth: 1
 
    testing/discussions/automated_vs_manual_test
+   testing/discussions/debugging-gui-tests-at-runtime
```

Comment:
We can discuss whether to have the section as a standalone or merge it with one of the existing ones.

Reply:
I am a bit torn on this.  I do like having things separated out into sections with a clear purpose, but when the content on individual pages is very small and that content may fit well elsewhere it can feel a bit like overkill.  
In this case the only section that this content may fit within aside from staying in its own section is event_loop_and_exceptions as debugging methods are meddling with the event loop.  

At this point though, I think I am in favor of leaving it as is, on its own page.